docs/developer/extending/apps/architecture/server-and-client-side-calls.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: Server and client side calls
+---
+
+Usually, apps are built on top of two communication methods - webhooks and API requests.
+While webhooks are always server-side (Saleor core calls the backend of your app),
+API requests from App to Saleor can be both server and client (frontend) sides.
+
+This article highlights main differences of both approaches.
+
+## Two types of tokens
+
+Saleor App API uses two types of tokens:
+
+- AppToken - used for server-side calls
+- AccessToken - used for client-side calls
+
+### AppToken
+
+AppToken is a one-time token generated when the App is being installed. Saleor will call `tokenTargetUrl` from the [App Manifest](../manifest.mdx) and App needs to store the token in the safe place,
+if it wants to consume the Saleor API. Official Saleor Apps and [app-sdk](../developing-apps/app-sdk.mdx) uses [APL concept](./apl.mdx) to store the token.
```

Comment:
just **typo**:
```suggestion
AppToken is a one-time token generated when the App is being installed. Saleor will call `tokenTargetUrl` from the [App Manifest](../manifest.mdx) and App needs to store the token in the safe place,
if it wants to consume the Saleor API. Official Saleor Apps and [app-sdk](../developing-apps/app-sdk.mdx) use [APL concept](./apl.mdx) to store the token.
```

**polish**:

```suggestion
AppToken is a one-time token generated during the App installation. To obtain it, Saleor will call `tokenTargetUrl` from the [App Manifest](../manifest.mdx). The token needs to be stored in a safe place, so the App can retrieve it to consume the Saleor API. Official Saleor Apps and [app-sdk](../developing-apps/app-sdk.mdx) use the [APL concept](./apl.mdx) to store the token.
```

Reply:
>To obtain it, Saleor will call `tokenTargetUrl` 

this suggests Saleor needs the token, no the app